rust/numaflow-core/src/shared/create_components.rs

```diff
@@ -334,8 +336,17 @@ pub async fn create_source(
                 None,
             ))
         }
-        SourceType::Serving(_) => {
-            unimplemented!("Serving as built-in source is not yet implemented")
+        SourceType::Serving(config) => {
+            let serving = ServingSource::new(Arc::clone(config), batch_size, read_timeout).await?;
```

Comment:
```suggestion
            let serving = ServingSource::new(config, batch_size, read_timeout).await?;
```

why clone, can't we give the ownership?

Reply:
The `match` is on a reference, so the `config` here is a `&Arc<Settings>`.